src/receivers/AwsLambdaReceiver.ts

```diff
@@ -171,7 +186,7 @@ export default class AwsLambdaReceiver implements Receiver {
         const signature = this.getHeaderValue(awsEvent.headers, 'X-Slack-Signature') as string;
         const ts = Number(this.getHeaderValue(awsEvent.headers, 'X-Slack-Request-Timestamp'));
         if (!this.isValidRequestSignature(this.signingSecret, rawBody, signature, ts)) {
-          this.logger.info(`Invalid request signature detected (X-Slack-Signature: ${signature}, X-Slack-Request-Timestamp: ${ts})`);
+          this.invalidRequestSignatureHandler({ rawBody, signature, ts });
           return Promise.resolve({ statusCode: 401, body: '' });
```

Comment:
Rather than just wrapping the logging part, the next line `Promise.resolve({ statusCode: 401, body: '' });` should be included in the callback too. Some developers may want to customize the status code, headeres, and body content. If we provide the callback for this, the pattern needs to be supported too like we do for other similar callbacks for HTTP/ExpressReceiver.

Reply:
Hi @seratch I updated a code. Can you have a look?